terasoluna-gfw-functionaltest-web/src/main/webapp/WEB-INF/views/logging/xTrackMDCPutFilter.jsp

```diff
@@ -21,7 +21,7 @@
 	<c:if test="${!(empty xTrackMDC)}">
 	${f:h(attributeName)}：<div id="xTrackMDC">${f:h(xTrackMDC)}</div>
 	</c:if>
-	<c:if test="${checkResponseHeaderXTrack}">
+	<c:if test="${!(empty checkResponseHeaderXTrack)}">

```

Comment:
I agreed with your fixes because most web articles use empty for both null and empty checking.